config/vectors.json

```diff
@@ -0,0 +1,60 @@
+{
+  "suites": {
+    "bbs-2023": {
+      "keyTypes": [["Bls12381G2", "BBS-BLS12-381-SHA-256"]],
```

Comment:
Why is this a nested array?

Reply:
Maybe let's bring it back when it's used?

The array inside of an array feels...weird/confusing.

Reply:
@BigBlueHat So I don't want to waste work so change it to an array of objects here:

https://github.com/w3c/vc-di-bbs-test-suite/pull/13/commits/dacdb3e419a16880846a866b5f28e954e36f5ebb

Reply:
Thanks! That reads better.